teku/src/main/java/tech/pegasys/teku/cli/subcommand/VoluntaryExitCommand.java

```diff
@@ -149,15 +153,31 @@ private void submitExitForValidator(final BLSPublicKey publicKey, final Integer
   }
 
   private Optional<Bytes32> getGenesisRoot() {
-    return apiClient.getGenesis().map(response -> response.getData().getGenesisValidatorsRoot());
+    try {
+      return apiClient.getGenesis().map(response -> response.getData().getGenesisValidatorsRoot());
+    } catch (RuntimeException ex) {
+      if (Throwables.getRootCause(ex) instanceof ConnectException) {
+        SUB_COMMAND_LOG.error(getFailedToConnectMessage());
+        System.exit(1);
+      }
+    }
```

Comment:
Would be good to avoid the duplication in this try catch.  Given it's a `RuntimeException` I wonder if we can just catch it higher up instead of in these private methods and handle it in one place?

Would also be kind of nice if the api client threw `UncheckedIOException` instead of `RuntimeException` so we don't have to catch something so generic.